api/services/SiteMembershipCreator.js

```diff
@@ -0,0 +1,55 @@
+const GitHub = require('./GitHub');
+const { Site, User } = require('../models');
+
+const checkGithubRepository = ({ user, owner, repository }) =>
+  GitHub.getRepository(user, owner, repository).then((repo) => {
+    if (!repo) {
+      throw {
```

Comment:
Looks like this condition is not covered by tests:

<img width="603" alt="screen shot 2017-10-30 at 9 47 59 am" src="https://user-images.githubusercontent.com/697848/32177156-87d115ca-bd57-11e7-8f33-0b828d7eaad7.png">

Reply:
Ah, good catch. I'll put one in there.

Reply:
Not sure if I mentioned it to you, but html coverage reports are written to your local `coverage/` directory. You can launch a static html server in that directory to explore the reports. Very helpful!